### PR TITLE
Interview post for Joni Trythall from YupGup

### PR DESCRIPTION
**Pull request by jcjudkins**
- Added blog post featuring Joni Trythall
- Added .jpg of Joni with DCUS 2023 background

### CODE REVIEW

File: _posts/2023-06-08-Joni-Trythall-YupGup.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+> For side projects, I’d suggest never letting visual design be a blocker to creating things and getting them out into the world. Everything can be redesigned or iterated upon later. This type of design is important, but it would be a shame for it to squash creative ideas and sharing. Do what you can with free/low-cost resources out there for now and don’t worry about making something perfectly pretty before showing others. Additionally, tools like Figma have taken away the learning curve barrier that Illustrator created when putting together basic logos and icons."
+
+## Any projects you would like to talk about?
+
+> Joni: "At YupGup we are kicking off a little initiative this summer and I’d love to share it here first with this community. It’s called “Ask a Designer” and it’s simply a friendly space for developers to ask designers questions. We want to help take the stigma, mystery, and intimidation out of design in general.  
+
+> You can email us at [askadesigner@yupgup.com](askadesigner@yupgup.com)–I can’t wait to chat."
```
Comment (drewbrew) [inline _posts/2023-06-08-Joni-Trythall-YupGup.md:61]: do you need a `mailto:` here?

File: _posts/2023-06-08-Joni-Trythall-YupGup.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+## Any projects you would like to talk about?
+
+> Joni: "At YupGup we are kicking off a little initiative this summer and I’d love to share it here first with this community. It’s called “Ask a Designer” and it’s simply a friendly space for developers to ask designers questions. We want to help take the stigma, mystery, and intimidation out of design in general.  
+
+> You can email us at [askadesigner@yupgup.com](askadesigner@yupgup.com)–I can’t wait to chat." 
+
+## I saw in your YupGup bio that you have a very interesting hobby. What got you into raising butterflies?
+
+> Joni: So like a lot of things, I initially showed interest in raising butterflies to get my son involved but it ended up with me taking it to a weird level. I ordered some TINY baby caterpillars from an organization called Monarch Watch to get started. We learned so much after that first year that it’s been easier and easier to attract them naturally and raise them from eggs. By prioritizing native plants, adding milkweed, and avoiding chemicals, we’ve been able to collect 30+ eggs a year and even got our yard certified as an official monarch waystation with these changes.
```
Comment (drewbrew) [inline _posts/2023-06-08-Joni-Trythall-YupGup.md:65]:
```suggestion
> Joni: So like a lot of things, I initially showed interest in raising butterflies to get my son involved but it ended up with me taking it to a weird level. I ordered some TINY baby caterpillars from an organization called [Monarch Watch](https://monarchwatch.org/) to get started. We learned so much after that first year that it’s been easier and easier to attract them naturally and raise them from eggs. By prioritizing native plants, adding milkweed, and avoiding chemicals, we’ve been able to collect 30+ eggs a year and even got our yard certified as an official [Monarch Waystation](https://monarchwatch.org/waystations/) with these changes.
```

Adding some links for context (and to support a good cause)

File: _posts/2023-06-08-Joni-Trythall-YupGup.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+>"I published resources (usually involving fruit) and conducted workshops as I learned it to help retain the knowledge better–like with [Pocket Guide to Writing SVG](https://svgpocketguide.com), [Flexbox Cheatsheet](https://jonitrythall.com/content/flexboxsheet.pdf), and [Tuts+ Town](https://webdesign.tutsplus.com/tutorials/web-design-for-kids-welcome-to-tuts-town--cms-23680)–and was eventually hired as a product designer. From there I spent time as a front-end developer, UX designer, marketing designer, and visual designer–with lots of learning on the job. So I’ve worn many different design hats over the years, but am happiest in the brand strategy space where I am now."
+
+## As a local of Durham, NC, I recognize the images you created really capture Durham. What did you use for inspiration?
+
+> Joni: "The inspiration was entirely the DCUS team. While I did some initial Googling, the magic really happened during our design kick-off call. Peter lives in Durham and excitedly listed so many wonderful things about the area–you can tell he really loves it there–and the rest was gauging everyone else’s reaction to the events and locations mentioned.
```
[the remark this answers is not in the document]
Comment (jcjudkins) [reply _posts/2023-06-08-Joni-Trythall-YupGup.md:34]: Good Catch! Thank you!

File: _posts/2023-06-08-Joni-Trythall-YupGup.md (hunk 1 above)
Comment (jcjudkins) [reply _posts/2023-06-08-Joni-Trythall-YupGup.md:61]: I do!

File: _posts/2023-06-08-Joni-Trythall-YupGup.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+> At the same time, though, we didn’t want to be locked into just one aspect of the area. In the end the theme took on more of a “lemur explores Durham” vibe to capture additional landmarks like the bull statue and the farmer’s market. 
+
+> I’ve never been to Durham but working on this project has put it on my bucket list." 
+
+
+## I had a much more artistic friend take a look at your work and help inform my questions. She thinks I should ask you about the “flat” art style. Any insight into what “flat” art style is and why you use it?
+
+> Joni: "I love this question! My first experiences with illustration were actually CSS-based drawings like this one [codepin.io/jonitrythall](https://codepen.io/jonitrythall/pen/zYGKEg)–I chose the crab as a hat tip for DCUS 2022 of course. Putting basic, flat, two dimensional shapes together was the easiest way to get a graphic in place with individual shapes for animating. This same approach carried over into Illustrator and learning about SVGs.
```
[the remark this answers is not in the document]
Comment (jcjudkins) [reply _posts/2023-06-08-Joni-Trythall-YupGup.md:45]: 👍

**Comment by jcjudkins** [comment]
Updated with all suggested changes!

**Review by drewbrew — APPROVED** [review]

**Comment by tim-schilling** [comment]
This is fantastic! Well done Jason on the interview and these are some really great answers from Joni. I definitely laughed at "mascot-loving community"

**Review by tim-schilling — APPROVED** [review]

**Pull request merged**